.circleci/config.yml

```diff
@@ -66,6 +66,8 @@ jobs:
           paths:
             - .
   backend:
+    environment:
+      - RAILS_ENV: development
```

Comment:
oh weird. I meant to say this was test, but backend tests are passing so maybe it's OK to just delete this piece?